lua/mason-core/installer/registry/util.lua

```diff
@@ -7,12 +7,24 @@ local platform = require "mason-core.platform"
 
 local M = {}
 
+---Helper functions for deprecated methods in nightly
+---@return function
+function M.islist()
+    return vim.fn.has "nvim-0.10" == 1 and vim.islist or vim.tbl_islist ---@diagnostic disable-line: deprecated
+end
```

Comment:
I'm also deprecating these functions in my code and using this as a reference. am I missing something here? or should this be:

```suggestion
---@param t? table
---@return boolean `true` if list-like table, else `false`.
function M.islist(t)
	return (vim.fn.has("nvim-0.10") == 1 and vim.islist or vim.tbl_islist)(t) ---@diagnostic disable-line: deprecated
end
```

or maybe just:

```suggestion
M.islist = vim.fn.has("nvim-0.10") == 1 and vim.islist or vim.tbl_islist
```